### PR TITLE
Feature/update services

### PR DESCRIPTION
**Pull request by jnation3406**
Adding you as a reviewer just to see the latest commits I made to fix zip downloading. You did most of the work so I'll review it too, but it should be working now. i did still have to run in a private browsing window to not have the /profile issue I thought you said you fixed, but maybe it is just some old cookies on my system for that.

### CODE REVIEW

File: nginx.conf [hunk 1]
```diff
@@ -57,4 +58,85 @@ http {
             proxy_pass http://minio;
         }
     }
+
+    # Currently, uncommenting even this breaks the nginx proxying for Minio
```
Comment (mgdaily) [inline nginx.conf]: Oops, forgot to remove these comments.

File: nginx.conf [hunk 2]
```diff
@@ -57,4 +58,85 @@ http {
...
+    # a few requests go through to the science archive and then nginx -T says:
+    #nginx: [emerg] host not found in upstream "minio1:9000" in /etc/nginx/nginx.conf:26
+    #nginx: configuration file /etc/nginx/nginx.conf test failed
+
+
+    # Need to get this working, then add the proxy endpoints for zip files
+    server {
+        listen 9500;
```
Comment (mgdaily) [inline nginx.conf:63]: Interesting, I swear I tried this to no success! Maybe it was the changing of the archive api to run on port 9501?

**Review by mgdaily — APPROVED** [review]

File: nginx.conf (hunk 1 above)
Comment (jnation3406) [reply nginx.conf]: oh no i think I forgot!

File: nginx.conf (hunk 2 above)
Comment (jnation3406) [reply nginx.conf:63]: yeah that might do it... but im not sure how it wouldve made minio unavailable, i think just the archiveapi wouldve been unavailable then.

**Pull request merged**